apps/fluent-tester/package.json

```diff
@@ -23,7 +23,10 @@
     "@fluentui-react-native/stack": "0.1.0",
     "@uifabricshared/theming-react-native": "0.7.15",
     "@uifabricshared/themed-stylesheet": "0.3.16",
-    "react-native-svg": "^11.0.0"
+    "react-native-svg": "^11.0.0",
+    "react": "16.11.0",
```

Comment:
If you only need this for testing you can get rid of them and have them just be devDependencies

Reply:
You are correct, removing them now